src/model/saveScore.js

```diff
@@ -0,0 +1,29 @@
+export const saveScore = (gameType, userName, userScore) =>{
+
+    let scoreTable = getScores(gameType);
+
```

Comment:
Te puste linijki są tutaj potrzebne?

Reply:
mi ułatwiają czytanie kodu, dlatego je sobie zostawiłem. imo jest to bardziej czytelne i łatwiej pozwala przeanalizować krok po kroku co sie dzieje w funkji :)

Reply:
Dla mnie wręcz przeciwnie. 
Zapytajmy @PiotrRynio czy tak może zostać.